spoti-vote-backend/server.js

```diff
@@ -34,6 +30,8 @@ function getInstanceById(instanceId) {
 	return instance;
 }
 
+
+//Dont know if i should also make this in the jsdocs format since technicly not functions
```

Comment:
should be documented like a REST-interface. E.g. [Spotify API](https://beta.developer.spotify.com/documentation/web-api/reference/playlists/change-playlist-details/)

Reply:
*every interface should be documented

Reply:
I did the best i could